.github/workflows/docker-image-common.yml

```diff
@@ -25,8 +29,7 @@ jobs:
     # Variables that might be needed by the calling workflow
     outputs:
       REGISTRY: ${{ env.REGISTRY }}
-      IMAGE_NAME: ${{ env.IMAGE_NAME }}
-      image_tag: ${{ steps.set-image-tag.outputs.IMAGE_TAG }}
+      IMAGE_NAME: ${{ steps.set-image-name.outputs.IMAGE_NAME }}
```

Comment:
Instead of defining IMAGE_NAME in the `env` block above, now we're getting it from the `set-image-name` step below. (`image_tag` is no longer needed.)